packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-conf

```diff
@@ -11,3 +11,11 @@ flags:
     passDeviceSpecs: {{default true settings.kubernetes.device-plugins.nvidia.pass-device-specs}}
     deviceListStrategy: {{default "volume-mounts" settings.kubernetes.device-plugins.nvidia.device-list-strategy}}
     deviceIDStrategy: {{default "index" settings.kubernetes.device-plugins.nvidia.device-id-strategy}}
+{{#if (gt settings.kubernetes.nvidia.device-plugins.max-sharing-per-gpu 1)}}
+sharing:
+  timeSlicing:
+    renameByDefault: {{settings.kubernetes.nvidia.device-plugins.rename-shared-gpu}}
```

Comment:
This isn't guaranteed to have a value just because `max-sharing-gpu` does:
```suggestion
  timeSlicing:
    renameByDefault: {{default false settings.kubernetes.nvidia.device-plugins.rename-shared-gpu}}
```